client/js/nonprofits/donate/info-step.js

```diff
@@ -38,8 +37,14 @@ function init(donation$, parentState) {
   state.showDedicationForm$ = flyd.map(()=> false, state.submitDedication$)
 
   // Save donor supporter record
-  state.supporterFields = supporterFields.init({required: {email: true}}, parentState.params$)
```

Comment:
Would you happen to know why you changed this from `parentState.params$` to `state.params$`?

Reply:
I don't see any reason why that would fail; it actually makes sense but I want to make sure there's no something I'm missing.

Reply:
I feel reasonably certain this shouldn't cause any issues so we'll go forward with it.